media/js/src/GraphViewer.jsx

```diff
@@ -87,664 +87,102 @@ export default class GraphViewer extends React.Component {
             const parsed = reader.parse(this.props.gInstructions);
             const instructions = writer.render(parsed);
 
-            instructionsEl = <p className="lead text-secondary" dangerouslySetInnerHTML={{__html:instructions}}></p>;
+            instructionsEl = <p className="lead" dangerouslySetInnerHTML={{__html:instructions}}></p>;
         }
 
-        let leftSide = null;
+        const commonViewerProps = {
+            isInstructor: isInstructor,
+            displayLabels: displayLabels,
+            displaySliders: displaySliders,
+            updateGraph: this.updateGraph,
+        };
+
+        let leftSide = (
+            <p>Loading...</p>
+        );
+        if (
+            typeof this.props.gType !== 'undefined' &&
+                this.props.gType !== null
+        ) {
+            leftSide = (
+                <JXGBoard
+                    id={'editing-graph'}
+                    width={BOARD_WIDTH}
+                    height={BOARD_HEIGHT}
+                    {...this.props}
+                />
+            );
+        }
         let rightSide = null;
 
         if (this.props.gType === 0 || this.props.gType === 9) {
             // Demand-Supply, possibly AUC (area under curve)
-            leftSide = <JXGBoard
-                           id={'editing-graph'}
-                           width={BOARD_WIDTH}
-                           height={BOARD_HEIGHT}
-                           shadow={!isInstructor}
```

Comment:
by removing the `shadow` prop from the JXGBoard, shadows are never present in the graph. These are used in the student views of the graph, see: https://econpractice.ctl.columbia.edu/course/6/graph/492/public/

![Screenshot_2022-11-23_10-59-46](https://user-images.githubusercontent.com/59292/203592307-10c7ca7c-ffd1-47c5-a2ab-260cb118b6aa.png)

I will add this back in here.

Reply:
I didn't notice that and I had tested for it. I still saw shadows while in student view. It's why I felt comfortable removing that line. If I remember I tried getting rid of it because `isInstructor` was giving me trouble.